client/components/mma/cancel/CancellationReasonSelection.tsx

```diff
@@ -70,16 +72,27 @@ const ReasonPicker = ({
 
 	return (
 		<>
-			<ProgressIndicator
-				steps={[
-					{ title: 'Reason', isCurrentStep: true },
-					{ title: 'Review' },
-					{ title: 'Confirmation' },
-				]}
-				additionalCSS={css`
-					margin: ${space[5]}px 0 ${space[12]}px;
-				`}
-			/>
+			{featureSwitches.supporterplusCancellationOffer &&
+			productType.productType === 'supporterplus' ? (
+				<ProgressStepper
+					steps={[{ isCurrentStep: true }, {}, {}, {}]}
```

Comment:
Just for my understanding - why don't any of these steps have a `title` property? (Like in the example on the other side of the ternary).

Reply:
The new designs for the cancellation flow don't have titles above each of the stages in the step indicator